tapdb/sqlc/queries/assets.sql

```diff
@@ -403,20 +406,24 @@ LIMIT 1;
 
 -- name: FetchGroupByGenesis :one
 SELECT
+    key_group_info_view.version AS version,
     key_group_info_view.tweaked_group_key AS tweaked_group_key,
     key_group_info_view.raw_key AS raw_key,
     key_group_info_view.key_index AS key_index,
     key_group_info_view.key_family AS key_family,
     key_group_info_view.tapscript_root AS tapscript_root,
-    key_group_info_view.witness_stack AS witness_stack
+    key_group_info_view.witness_stack AS witness_stack,
+    key_group_info_view.custom_subtree_root AS custom_subtree_root
 FROM key_group_info_view
 WHERE (
     key_group_info_view.gen_asset_id = @genesis_id
 );
 
 -- name: QueryAssets :many
 SELECT
-    assets.asset_id AS asset_primary_key, assets.genesis_id, version, spent,
+    assets.asset_id AS asset_primary_key,
+    assets.genesis_id, assets.version,
+    spent,
```

Comment:
ultra-nit: What are our coding style guidelines re line breaks and such for SQL?

Reply:
Good question. I'm not sure. @guggero do we have a doc on this?

I would think that if we use `AS`, example `assets.asset_id AS asset_primary_key,` then that gets its own line. And otherwise we wrap at column 80.

So I've moved `spent,` to the line above.

Reply:
> I would think that if we use AS, example assets.asset_id AS asset_primary_key, then that gets its own line. And > otherwise we wrap at column 80.

> So I've moved spent, to the line above.

There's no style guide yet. But what you describe above matches what I normally do.